src/components/WebexMeetingControlBar/WebexMeetingControlBar.jsx

```diff
@@ -125,6 +130,9 @@ export default function WebexMeetingControlBar({
         type={key}
         meetingID={meetingID}
         showText={controlTexts[i]}
+        autoFocus={i === firstFocusableControl(isActive, names) || false}
+        tabIndex={i === firstFocusableControl(isActive, names) ? 1 : i + 2}
+        ariaLabel={key === 'join-meeting' ? `${audioStatus}, ${videoStatus}` : null}
```

Comment:
the control bar should not treat some controls specially. Can this be implemented in the join control?

Reply:
and add the word "labels" to the commit message please, "add accessibility labels for meeting controls"

feat(WebexMeetingControl)